### PR TITLE
Add Advanced Go raffle page

### PR DESCRIPTION
**Pull request by myitcv**
screenshot

### CODE REVIEW

File: content/advanced-go/index.md [hunk 1]
```diff
@@ -0,0 +1,21 @@
+---
+title: "Advanced Ultimate Go Raffle"
+date: 2019-05-09T15:28:25+01:00
```
Comment (thoeni) [inline content/advanced-go/index.md]: I guess this doesn't matter?

File: content/advanced-go/index.md [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+---
+
+## Advanced Ultimate Go Raffle!
+
+Enter the London Gophers Advanced Ultimate Go raffle to win a place on this [Ardan Labs](https://www.ardanlabs.com/)
```
Comment (thoeni) [inline content/advanced-go/index.md]: Maybe in quotes?
```suggestion
Enter the London Gophers "Advanced Ultimate Go" raffle to win a place on this [Ardan Labs](https://www.ardanlabs.com/)
```

File: content/advanced-go/index.md [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+course! For full details of the course are available
```
Comment (thoeni) [inline content/advanced-go/index.md]: I'd say either:
```suggestion
course! Full details of the course are available
```
or
```suggestion
course! For full details of the course please see
```

**Review by thoeni — APPROVED** [review]
Looks good, a couple of comments, but feel free to ignore them if they don't make sense, more like suggestions - and I might be wrong!

File: content/advanced-go/index.md (hunk 1 above)
Comment (myitcv) [reply content/advanced-go/index.md]: Fixed.

File: content/advanced-go/index.md (hunk 2 above)
Comment (myitcv) [reply content/advanced-go/index.md]: Done.

File: content/advanced-go/index.md (hunk 3 above)
Comment (myitcv) [reply content/advanced-go/index.md]: Thanks - took your first suggestion.

**Comment by myitcv** [comment]
Will merge after passing build.

**Pull request merged**